force-app/main/default/classes/GiftBatchForQueueable_TEST.cls

```diff
@@ -0,0 +1,439 @@
+        // Arrange
+        Id dummyDataImportBatchId = UTIL_UnitTestData_TEST.mockId(DataImportBatch__c.SObjectType);
+        GiftBatchForQueueable dummyGiftBatchForQueueable = buildGiftBatch(dummyDataImportBatchId);
+        List<DataImport__c> authorizedGift = new List<DataImport__c> {
+            new DataImport__c(
+                Id = UTIL_UnitTestData_TEST.mockId(DataImport__c.SObjectType),
+                Payment_Status__c = ElevateTransactionStatus.AUTHORIZED.name(),
+                Status__c = BDI_DataImport_API.bdiDryRunValidated
+            )
+        };
+
+        GiftsMock giftsMock = new GiftsMock().with(authorizedGift);
+        dummyGiftBatchForQueueable.gifts = stubFor(giftsMock);
+
+        ElevateBatchServiceMock elevateBatchServiceMock = new ElevateBatchServiceMock();
+        dummyGiftBatchForQueueable.elevateBatchService = stubFor(elevateBatchServiceMock);
+
+        PS_IntegrationServiceConfigMock integrationServiceConfigMock = new PS_IntegrationServiceConfigMock(false);
+        dummyGiftBatchForQueueable.paymentIntegrationServiceConfig = stubFor(integrationServiceConfigMock);
+
+        // Act
+        dummyGiftBatchForQueueable.captureElevateBatches();
+
+        // Assert
+        System.assertEquals(false, elevateBatchServiceMock.enqueuePaymentCaptureJobForCalled);
+        System.assertEquals(false, giftsMock.authorizedToPaymentProcessingStatusWasCalled);
+    }
+
+    @IsTest
+    static void shouldReturn14ChunksOfGiftsWhen700GiftsAreProcessable() {
+        // Arrange
+        Id dummyDataImportBatchId = UTIL_UnitTestData_TEST.mockId(DataImportBatch__c.SObjectType);
+        GiftBatchForQueueable dummyGiftBatchForQueueable = buildGiftBatch(dummyDataImportBatchId);
+
+        // These gifts cannot move to processing status
+        List<DataImport__c> importedGifts =
+            buildGifts(5, BDI_DataImport_API.bdiImported, dummyDataImportBatchId, true);
+        List<DataImport__c> processingGifts =
+            buildGifts(4, BDI_DataImport_API.bdiProcessing, dummyDataImportBatchId, true);
+
+        // These gifts can move to processing status
+        List<DataImport__c> failedGifts =
+            buildGifts(200, BDI_DataImport_API.bdiFailed, dummyDataImportBatchId, true);
+        List<DataImport__c> dryRunValidatedGifts =
+            buildGifts(500, BDI_DataImport_API.bdiDryRunValidated, dummyDataImportBatchId, true);
+
+        GiftsSelectorForProcessingMock selectorMock =
+            new GiftsSelectorForProcessingMock()
+                .withImportedGifts(importedGifts)
+                .withProcessingGifts(processingGifts)
+                .withFailedGifts(failedGifts)
+                .withDryRunValidatedGifts(dryRunValidatedGifts);
+        dummyGiftBatchForQueueable.giftsSelector = stubFor(selectorMock);
+
+        // Act
+        dummyGiftBatchForQueueable.chunkGiftsThatCanBeProcessed();
+
+        // Assert
+        System.assertEquals(14, dummyGiftBatchForQueueable.chunkedIds.size(),
```

Comment:
Is 14 the correct number here because the batch is defaulted to a processing size of 50? If so cannot be explicitly called out here or somewhere in this test? Maybe with just a comment?